cumulusci.yml

```diff
@@ -136,12 +142,6 @@ flows:
                     managed: True
             7:
                 task: add_picklist_values
```

Comment:
I can't tell if this is being added or removed of untouched, but do we need to run add_picklist_values as part of the uninstall_packaged_incremental? (If I am honest, I just barely understand what uninstall_packaged_Incremental does and why

Reply:
so @jstvz correct me if im wrong but uninstall_packaged_incremental removes the metdata in source that has been deleted in an org.

Reply:
Vice versa, @jofsky. `uninstall_packaged_Incremental` will remove any metadata in the org not in `src/`.

That said, we aren't touching `uninstall_packaged_Incremental` here.